PLAN.md

```
@@ -0,0 +1,44 @@
+# Canonical Plan of MagnetarQuill
+
+This plan captures milestones, tasks, effort estimates, and task status; its structure must be kept intact.
+
+## Milestones Overview
+| Milestone ID | Name | Target Date | Description | Completion Criteria |
```

markdownlint-cli2 (0.20.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)